data-raw/who.csv

```diff
@@ -4368,40 +4368,40 @@ Myanmar,MM,MMR,2010,106,3043,6578,6688,5607,3632,2308,196,2452,3454,2752,2525,18
-Namibia,"NA",NAM,2001,20,322,993,732,318,150,116,32,394,729,404,168,91,66,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2002,19,301,1033,750,326,146,96,42,357,795,484,182,91,67,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2003,31,364,1109,838,419,196,108,47,451,927,571,216,108,102,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2004,31,319,1092,866,371,159,131,30,400,819,554,203,106,74,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2005,98,355,1027,874,365,146,120,105,399,809,525,213,95,91,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2006,86,347,1052,799,386,174,146,74,485,875,521,239,92,80,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2007,57,370,1018,786,346,149,120,69,417,826,513,242,102,76,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2008,30,387,1033,757,346,149,132,73,466,702,437,226,110,80,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2009,41,357,936,689,348,166,121,84,384,678,407,214,97,86,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2010,36,359,852,680,287,146,126,67,429,685,382,206,122,87,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2011,48,337,844,660,361,152,138,78,427,653,410,185,100,110,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2012,61,358,810,686,292,157,137,81,394,582,396,198,84,97,386,153,477,516,268,162,135,339,200,379,306,183,98,90,180,126,228,261,138,46,55,163,116,219,177,84,25,47,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
-Namibia,"NA",NAM,2013,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,603,612,1496,1404,723,392,309,491,688,1137,747,408,220,202
+Namibia,NA,"NAM",1980,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
```

Comment:
This is normal?

Reply:
I suspect this is just a change in how readr writes NA; I think were some changes around whether `"NA"` was interpreted as missing or not.

Reply:
It also struck me as weird that `"NAM"` started getting quoted all of a sudden

Reply:
Yeah, looks like maybe a small readr bug: https://github.com/tidyverse/readr/issues/1386